### DISCUSSION TITLE
Welcome to AS_A333_P3D_Patch Discussions!

### DISCUSSION THREAD

**Discussion by GPUthesteve** [GPUthesteve/AS_A333_P3D_Patch#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Tinosuke** [comment]
ไฟล์ .air น่าจะมีปัญหาครับ เข้าแล้ว CTD ตลอดเลยถ้าไม่ทับไฟล์ .air ยังเล่นได้ครับ

**Reply by GPUthesteve** [reply]
ถ้าอย่างนั้น ลองใช้ airfile เดิม บินรูทเดิมดูครับ (drag model เดิมๆ มันจะแย่หน่อยนะครับ หากมีพวกโปรแกรมแก้ airfile อย่าง AirEd รบกวนแก้ Entry 1101 Aerodynamics / part CD0 Zero lift drag coeff เป็นสัก 35 ก็พอครับ ไม่ก็ส่งตัวเดิมมาให้ผมก่อนก็ได้ เดี๋ยวปรับจากตัวนั้นเอาครับ)

ส่วนตัวผมบินทดสอบแค่ 4-6 ชม อาจจะไม่เห็นผลครับ ต้องขออภัย เนื่องจากช่วงนี้ง่วนกับ Aerosoft อีกตัวอยู่ครับ (อุบไว้ก่อน)
